### PR TITLE
test branch

### PR DESCRIPTION
**Pull request by ja-son**
## Test Branch
this is a test PR

### CODE REVIEW

**Comment by ja-son** [comment]
I'm concerned the CI/CD build is failing

**Comment by big-dubs** [comment]
It doesn't look like you made any real changes

**Review by big-dubs — COMMENTED** [review]
All you did was remove a comment?

File: next.config.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-// import our database logic
```
Comment (ja-son) [inline next.config.js:1]: Well yeah, the comment was not needed.

Comment (big-dubs) [reply next.config.js:1]: Ok I see your point

**Comment by ja-son** [comment]
Not accepting this into master.

**Pull request closed without merging**